src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Options } from './types/Options';
+import { getTodos } from './api';
+
+function getFilteredTodos(todos: Todo[], filter: Filter) {
+  // let filteredTodos = [...todos];
+
+  const filteredTodos: Todo[] = todos.filter(todo => {
+    switch (filter.option) {
+      case Options.active:
+        return !todo.completed;
+
+      case Options.completed:
+        return todo.completed;
+
+      case Options.all:
+      default:
+        return filteredTodos;
+    }
+  });
+
+  if (filter.query) {
+    filteredTodos.filter(({ title }) => title.includes(filter.query.trim()));
+  }
+
+  return filteredTodos;
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [showModal, setShowModal] = useState(false);
```

Comment:
Fix all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isShowModal, setIsShowModal] = useState(false);
```